.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Chromatic'
+
+on: push
+
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
Please use **actions/checkout@v3** if you can

Reply:
I changed it to v3